Assignment/Program.cs

```diff
@@ -1,7 +1,25 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
-      }
+﻿//Display the individual digits of a given number.
+Console.Write ("Give decimal value to split : ");
+string test = Console.ReadLine () ?? "";
+if (double.TryParse (test, out double d) && d > 0)

```

Comment:
Is the input/output right?

![image](https://github.com/YoganathanM/Assignment/assets/16486802/1c575f2c-9039-491d-81d4-8be5f0346a7f)